e2e/test/iothub/service/DevicesClientE2ETests.cs

```diff
@@ -201,48 +201,29 @@ public async Task DevicesClient_UpdateDevicesAsync_Works()
         {
             // arrange
 
-            var device1 = new Device(_idPrefix + Guid.NewGuid());
-            var device2 = new Device(_idPrefix + Guid.NewGuid());
-            var edge = new Device(_idPrefix + Guid.NewGuid());
-            using var serviceClient = new IotHubServiceClient(TestConfiguration.IotHub.ConnectionString);
+            IotHubServiceClient serviceClient = TestDevice.ServiceClient;
+            await using TestDevice testDevice1 = await TestDevice.GetTestDeviceAsync(_idPrefix).ConfigureAwait(false);
+            await using TestDevice testDevice2 = await TestDevice.GetTestDeviceAsync(_idPrefix).ConfigureAwait(false);
```

Comment:
Why dropping `+ Guid.NewGuid()` from device ID while calling `GetTestDeviceAsync`?

Reply:
TestDevice.GetTestDeviceAsync will add the GUID.